### ISSUE TITLE
INTRO HD not properly getting inverted

### ISSUE THREAD

**Issue by creativrss** [darkreader/darkreader#12201]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Pretty much the title

### Website Address

https://intro-hd.net/bigfilms-environments-pack-vol-1-2/

### Steps To Reproduce

Just by simply visiting the site.

### Screenshots

![image](https://github.com/darkreader/darkreader/assets/90452541/5e53e551-b7f8-432f-a44e-d7a061599fee)

### Operating System

MacOS

### Web Browser name and version

Brave: 1.61.120 Chromium: 120.0.6099.234 (Official Build) (x86_64) 

### Dark Reader version

4.9.76

### Additional Context

Not needed.

**Comment by Myshor** [comment]
Can you check other browser than Brave too?
I don't have anything with MacOS but I can not reproduce using Firefox or any Chromium based browser using Windows 11.

So it would be nice to check if it's Brave only bug or mayby MacOS related.

**Comment by creativrss** [comment]
> Can you check other browser than Brave too? I don't have anything with MacOS but I can not reproduce using Firefox or any Chromium based browser using Windows 11.
> 
> So it would be nice to check if it's Brave only bug or mayby MacOS related.

Yeah I guess it's MacOS, I checked with Firefox and Opera too. Still looks the same.
